2.4/examples/replica/mongodb-clustered.json

```diff
@@ -94,7 +94,13 @@
                 "command": [
                   "run-mongod", "initiate"
                 ],
-                "containerName": "mongodb"
+                "containerName": "mongodb",
+                "env": [
+                  {
+                    "name": "MONGODB_NODES_TO_WAIT",
```

Comment:
is this only for the initial template instantiation? or user will have to modify this every time the "replicas" value is changed?